src/types/markers.js

```diff
@@ -28,29 +63,34 @@ export type MarkerDisplayLocation =
   | 'marker-table'
   | 'timeline'
   // In the timeline, this is a section that breaks out markers that are related
-  // to memory.
+  // to memory. When memory counters are enabled, this is its own track, otherwise
+  // it is displayed with the main thread.
   | 'timeline-memory'
+  // TODO - This is not supported yet.
   | 'stack-chart';
 
 export type MarkerSchema = {
   // The unique identifier for this marker.
-  name: string, // e.g. "GCMajor-complete"
+  name: string, // e.g. "CC"
 
   // The label of how this marker should be displayed in the UI.
-  // TODO - It would be nice to support printf-like features here
-  label: string, // e.g. "GCMajor"
+  // If none is provided, then the name is used.
+  label?: string, // e.g. "Cycle Collect"
```

Comment:
Good idea to make it optional so we can just fall back to name/key fields instead.